rules/linux/execution_cpulimit_binary.toml

```
@@ -26,9 +26,7 @@ timestamp_override = "event.ingested"
 type = "eql"
 
 query = '''
-sequence by host.id, process.pid with maxspan=1m
-[process where event.type == "start" and process.name == "cpulimit" and process.args == "-f" and process.args : ("/bin/sh", "/bin/bash", "/bin/dash", "sh", "bash", "dash")]
-[process where process.parent.name == "cpulimit" and process.name : ("bash", "sh", "dash")]
+process where event.type == "start" and process.name in ("bash", "sh", "dash") and process.parent.name == "cpulimit" and process.parent.args == "-f" and process.parent.args in ("/bin/sh", "/bin/bash", "/bin/dash", "sh", "bash", "dash") 
```

format only suggestion

```suggestion
process where event.type == "start" and process.name in ("bash", "sh", "dash") and 
  process.parent.name == "cpulimit" and process.parent.args == "-f" and 
  process.parent.args in ("/bin/sh", "/bin/bash", "/bin/dash", "sh", "bash", "dash") 
```